lib/3scale/backend/worker.rb

```diff
@@ -74,11 +74,11 @@ def perform(job)
       end
 
       def register_worker
-        redis.sadd(:workers, self)
+        redis.sadd(:workers, self.class.name)
```

Comment:
:thinking: wouldn't this register all workers under the same name? Isn't the point here to be able to identify registered workers? For example `self.to_s` may result in `worker-backend-pod-10-10...` while `self.class.name` would be all the same for any pod.

Reply:
Right, my bad. Fixed at https://github.com/3scale/apisonator/pull/384/commits/c1048fb2cf3564ce9d1e7502596daf7359dc1b8a